static/styles/compose.css

```diff
@@ -282,8 +285,9 @@
 }
 
 .compose_invite_user p,
+.compose_resolved_topic p,
 .compose_not_subscribed p {
-    margin: 0 20px;
+    margin: 5px 20px;
```

Comment:
@alya Can you check the alignment again?
![image](https://user-images.githubusercontent.com/59444243/148331902-04a99983-0d90-483a-99e8-d47b2aeacff8.png)
I have changed the top margin to fix the alignment.

Reply:
Cool, can you also post a screenshot with a narrow window to make sure it looks OK if the banner text takes up two lines?

Also, I don't know if this is a new issue or an existing one, but it looks like the text in the button might not be vertically centered like it should be.

Reply:
For two lines
![image](https://user-images.githubusercontent.com/59444243/148349023-5cad983b-8580-425d-8dd7-6d40fd0bdbcf.png)

Reply:
Ah, the "x" button is definitely in the wrong place in the 2-line version.

Reply:
(This is probably a pre-existing problem as well; we may have an issue for it.)

Reply:
I am stuck while working on fixing this cross button alignment issue from a long time. So I have open an issue #20839 for it.